src/Workspaces/Core/Portable/Simplification/QualifyMemberAccessCodeAction.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+
+namespace Microsoft.CodeAnalysis.Simplification
+{
+    internal class QualifyMemberAccessCodeAction : CodeAction.DocumentChangeAction
```

Comment:
Note: This isn't accessed from anywhere but AbstractQualifyMemberAccessCodeFixprovider.cs, so it could be a private type within `AbstractQualifyMemberAccessCodeFixprovider`